Make target .install is broken

```
$ python -m venv venv
$ source venv/bin/activate
$ make deps
$ make .install
pip install -e .[test]
Obtaining file:///Users/ddriddle/src/github.com/techservicesillinois/awscli-login
  Preparing metadata (setup.py) ... error
  error: subprocess-exited-with-error
  
  × python setup.py egg_info did not run successfully.
  │ exit code: 1
  ╰─> [7 lines of output]
      Traceback (most recent call last):
        File "<string>", line 2, in <module>
        File "<pip-setuptools-caller>", line 34, in <module>
        File "/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/setup.py", line 8, in <module>
          with open(path.join(here, 'docs/readme.rst'), encoding='utf-8') as f:
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
      FileNotFoundError: [Errno 2] No such file or directory: '/Users/ddriddle/src/github.com/techservicesillinois/awscli-login/docs/readme.rst'
      [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: metadata-generation-failed

× Encountered error while generating package metadata.
╰─> See above for output.

note: This is an issue with the package mentioned above, not pip.
hint: See above for details.

[notice] A new release of pip available: 22.3.1 -> 23.3.1
[notice] To update, run: pip install --upgrade pip
make: *** [Makefile:136: .install] Error 1
```

--- a/Makefile
+++ b/Makefile
@@ -132,7 +132,7 @@ coverage: .coverage
 develop: lint static .install develop-coverage
 
 # Install package in develop mode
-.install: setup.py
+.install: docs/readme.rst setup.py
 	pip install -e .[test]
 	@touch $@
 

